doc/classes/Particles2D.xml

```diff
@@ -286,16 +286,16 @@
 	</methods>
 	<members>
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
-			Number of particles to emit.
+			Number of particles that can exist simultaneously and that are emitted in one emission cycle.
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
 			Particle draw order. Uses [code]DRAW_ORDER_*[/code] values. Default value: [code]DRAW_ORDER_INDEX[/code].
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
-			If [code]true[/code] particles are being emitted. Default value: [code]true[/code].
+			Emits particles if [code]true[/code]. Default value: [code]true[/code].
```

Comment:
"Emits particles" is the wording for a method. Member variables do not directly take an action. The first wording is correct.